cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -68,9 +68,8 @@ int32_t cros_gralloc_driver::init()
 	char *node;
 
 	const int render_num = 10;
-	const int name_length = 50;
 	int node_fd[render_num];
-	char node_name[render_num][name_length] = { "" };
```

Comment:
can we define it as
char node_name[render_num][name_length]={{0}};

Reply:
@feijiang1 The coverity issue is as follows:
26. fixed_size_dest: You might overrun the 50-character fixed-size string node_name[availabe_node] by copying version->name without checking the length.
So I made this changes

Reply:
got it, if we use malloc, we have to free